client/modules/food/components/inventory/FoodItems.js

```diff
@@ -49,6 +52,11 @@ class FoodItems extends React.Component {
       this.setState({
         showModal: 'Edit',
         editModalFood: food,
+        initialModalInputFields: {
```

Comment:
Could just define a variable for setting modalInputFields and initialModalInputFields so you don't have to write it twice

Reply:
can you please elaborate this part further?

Reply:
just like this, so if you add/remove a field you don't have to change it in two places:
```js
const inputFields = {...}
this.setState({
   ...
   initialModalInputFields: inputFields,
   modalInputFields: inputFields
})
```

Reply:
I see! Thanks so much!